.github/workflows/nightly-release.yml

```diff
@@ -2,6 +2,7 @@ name: release-nightly
 on:
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch: {}
```

Comment:
```suggestion
  workflow_dispatch:
```
If I understand https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration correctly 🤷‍♂️

Reply:
![](https://media.giphy.com/media/3WmWdBzqveXaE/source.gif)

Reply:
I think the `{}` is optional but recommended in yaml?

Reply:
I recommend not using yaml at all 😂